index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Initial HTML file</title>
+</head>
```

Comment:
- [OPTIONAL] In order for the CSS file to apply the styling to the elements in the HTML file, the CSS file should be imported in the `<head>` section. It would be nice if you import your `style.css` here so we can see the color of the heading change.